### DISCUSSION TITLE
Enable swaggerUi feature for docker images

### DISCUSSION THREAD

**Discussion by kam1kaze** [grafana/grafana#47811] [General]
Hi,

I would like to see the enabled swaggerUi endpoint in official docker images https://hub.docker.com/r/grafana/grafana or at least add a separate docker tag with such a feature. If you agree and need just a PR, just let me know.

Thanks

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
